app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Place all arguments including `self` on a new line for `__init__` of all classes.

Reply:
fixed in [commit](https://github.com/job-artem/py-car-wash-station/commit/c7a74045fb18c03c88e7633921be4097e5f9e22c)